### PR TITLE
[Demo]: Updated e2e tests with changes from template-infra

### PR DESCRIPTION
**Pull request by rylew1**
## Ticket

Resolves #{TICKET NUMBER OR URL}

## Changes

- Test that changes made on `template-infra` repo still work

## Context for reviewers

This PR still appears to work on this repo:
- `template-infra` PR - https://github.com/navapbc/template-infra/pull/694
- original working PR on this repo: https://github.com/navapbc/platform-test-nextjs/pull/85

## Preview environment
- Service endpoint: http://p-86-app-dev-1797299030.us-east-1.elb.amazonaws.com
- Deployed commit: d3fc9afa8e738b176232f46804722f1cbdc9be4f

### CODE REVIEW

File: .github/workflows/e2e-tests.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          node-version: '20'
+
+      - name: Install Playwright browsers
+        run: make e2e-setup-ci
+
+      - name: Run e2e tests
+        run: make e2e-test APP_NAME=${{ inputs.app_name }} BASE_URL=${{ inputs.service_endpoint }}
+        env:
+          BASE_URL: ${{ inputs.service_endpoint }}
+          APP_NAME: ${{ inputs.app_name }}
```
Comment (lorenyu) [inline .github/workflows/e2e-tests.yml:38]: nit: delete, doesn't look like this is used anywhere

File: .github/workflows/e2e-tests.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+jobs:
+  e2e:
+    name: " " # GitHub UI is noisy when calling reusable workflows, so use whitespace for name to reduce noise
+    runs-on: ubuntu-latest
+
+    env:
+      BASE_URL: ${{ inputs.service_endpoint }}
+      APP_NAME: ${{ inputs.app_name }}
+
```
Comment (lorenyu) [inline .github/workflows/e2e-tests.yml:21]: nit: delete, doesn't look like this is used anywhere. in the "run e2e tests" step, you use inputs.app_name and inputs.service_endpoint, not env.app_name and env.service_endpoint

File: Makefile [hunk 3]
```diff
@@ -222,6 +222,28 @@ release-image-name: ## Prints the image name of the release image
...
+	libgstreamer-plugins-base1.0-0 libgstreamer-plugins-good1.0-0 libharfbuzz-icu0 libhyphen0 \
+	libenchant-2-2 libflite1 libgles2 libx264-dev
+	cd e2e && npm ci
+	cd e2e && npx playwright install --with-deps
+
+
+e2e-test: ## Run end-to-end tests
+	@:$(call check_defined, APP_NAME, You must pass in a specific APP_NAME)
+	@:$(call check_defined, BASE_URL, You must pass in a BASE_URL)
+	@cd e2e && cd $(APP_NAME) && APP_NAME=$(APP_NAME) BASE_URL=$(BASE_URL) npx playwright test $(E2E_ARGS)
```
Comment (lorenyu) [inline Makefile:245]: nit:

```suggestion
	@cd e2e/$(APP_NAME) && APP_NAME=$(APP_NAME) BASE_URL=$(BASE_URL) npx playwright test $(E2E_ARGS)
```

**Review by lorenyu — COMMENTED** [review]
LGTM, left a comment about getting rid of unneeded env vars, try that and test to see if it works

File: .github/workflows/e2e-tests.yml (hunk 2 above)
Comment (rylew1) [reply .github/workflows/e2e-tests.yml]: Thanks - looks like it worked:
```
Run make e2e-test APP_NAME=app BASE_URL=http://p-86-app-dev-1797299030.us-east-1.elb.amazonaws.com/
  make e[2](https://github.com/navapbc/platform-test-nextjs/actions/runs/10222286732/job/28286717255?pr=86#step:5:2)e-test APP_NAME=app BASE_URL=http://p-86-app-dev-17972990[3](https://github.com/navapbc/platform-test-nextjs/actions/runs/10222286732/job/28286717255?pr=86#step:5:3)0.us-east-1.elb.amazonaws.com
  shell: /usr/bin/bash -e {0}

Running [6](https://github.com/navapbc/platform-test-nextjs/actions/runs/10222286732/job/28286717255?pr=86#step:5:7) tests using 1 worker
······
  6 passed (8.3s)
  ```

**Review by lorenyu — APPROVED** [review]

**Comment by rylew1** [comment]
See the final PR on the `template-infra` repo - https://github.com/navapbc/template-infra/pull/694/files

Closing this

**Pull request closed without merging**